web/src/activity_ui.ts

```diff
@@ -59,6 +59,18 @@ export function clear_for_testing(): void {
     user_filter = undefined;
 }
 
+function update_status_circles(): void {
+    $(".user_circle").each(function () {
+        const user_id = Number.parseInt($(this).attr("data-status-sync-user-id") ?? "", 10);
+        if (Number.isNaN(user_id)) {
+            // This eliminates elements having more than one user-id.
+            return;
+        }
```

Comment:
There shouldn't be any, right? So you might want an assert here.

Reply:
Done